packages/itwin/ec3-widget/README.md

```diff
@@ -0,0 +1,47 @@
+# @itwin/ec3-widget-react
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+The EC3 widget is a UI component for iTwin Viewer applications that simplifies how users (and developers) interface with the [Carbon Calculation EC3 APIs](https://developer.bentley.com/apis/carbon-calculation/overview/).
```

Comment:
Why are we creating a new widget for every single carbon calculation integration?
We should've only had one pkg with multiple integrations in it.